sunpy/data/_sample.py

```diff
@@ -51,52 +65,58 @@
     "NORH_LIGHTCURVE": ("tca110810", "")
 }
 
-sample_files = {}
-for key in _files:
-    sample_files[key] = os.path.abspath(os.path.join(sampledata_dir, _files[key][0]))
+
+free_required=145227776
+if disk_usage('/').free > free_required :
```

Comment:
this may have problem on not unix-like machines, and this 'free space' on the `root` directory does not reflect that the user don't have a quota in the home directory.